### PR TITLE
Advisory - doctrine - CVE-2015-5723

### PR DESCRIPTION
**Pull request by Ocramius**
See http://www.doctrine-project.org/2015/08/31/security_misconfiguration_vulnerability_in_various_doctrine_projects.html - CVE-2015-5723

### CODE REVIEW

File: doctrine/annotations/2015-08-31.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+title:     Security Misconfiguration Vulnerability in various Doctrine projects
+link:      http://www.doctrine-project.org/2015/08/31/security_misconfiguration_vulnerability_in_various_doctrine_projects.html
+cve:       CVE-2015-5723
+branches:
+    1.0.x:
```
Comment (Ocramius) [inline doctrine/annotations/2015-08-31.yaml]: Do non-existing branches make any difference? I couldn't find any way to match ranges that are not covered by a branch anymore.

File: doctrine/annotations/2015-08-31.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+        time:     2015-08-31 14:22:00
```
Comment (Ocramius) [inline doctrine/annotations/2015-08-31.yaml]: I'll have to check for the tag date first.

File: doctrine/annotations/2015-08-31.yaml (hunk 1 above)
Comment (stof) [inline doctrine/annotations/2015-08-31.yaml]: no it does not. You could even make a single range ``<1.2.7`` in the 1.2.x branch

Comment (Ocramius) [inline doctrine/annotations/2015-08-31.yaml]: Gonna do that now then.

Comment (Ocramius) [inline doctrine/annotations/2015-08-31.yaml]: Much better.

**Comment by Ocramius** [comment]
@fabpot thanks!

**Pull request merged**